### PR TITLE
{devel}[GCCcore/7.3.0] Bazel v0.16.0

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Bazel/Bazel-0.16.0-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+version = '0.16.0'
+
+homepage = 'http://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably. 
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bazel/Bazel-0.16.0-GCCcore-7.3.0.eb]: @reedts Please include a SHA256 checksum (`--inject-checksums` will do this for you)

Comment (reedts) [reply easybuild/easyconfigs/b/Bazel/Bazel-0.16.0-GCCcore-7.3.0.eb]: Oops, my bad. Updated.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3a26fadf464f2bcd7154b238fd60009f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node3150.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/cff21c8a8053cb28d0ef2eabc6979d4c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/09afa3b9e19ff5680498da9c74c5e751 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @reedts!

**Pull request merged**